### PR TITLE
Mobile viewport

### PR DESCRIPTION
**Pull request by eddypjr**
### Types of Changes
- [x]  Added function undoRedoKey(RightContainer.tsx line 209) to keybind Undo and Redo actions
- [ ] Works for both Mac and Windows

### Purpose
- To allow users to Undo and Redo actions with ease.

### Tutorial
**Mac**
Undo: Command + z
Redo: Command + shift + z

**Windows:**
Undo: Control + z
Redo: Control + shift + z

### Screenshots
Screen Shot 2021-03-11 at 11 32 29 AM

### CODE REVIEW

File: app/src/reducers/componentReducer.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
 import initialState from '../context/initialState';
 import generateCode from '../helperFunctions/generateCode';
 import manageSeparators from '../helperFunctions/manageSeparators';
+import { isObjectType } from 'graphql';
```
Comment (khuongdn16) [inline app/src/reducers/componentReducer.ts:11]: Can you share the reason for this import ? I don't think this file needs grapql functions.

File: app/src/containers/RightContainer.tsx [hunk 2]
```diff
@@ -205,6 +206,18 @@ const redoAction = () => {
...
+const undoRedoKey = useCallback((e) => {
+  (e.key === 'z' && e.metaKey && !e.shiftKey) ? undoAction() :
+  (e.shiftKey && e.metaKey && e.key === 'z') ? redoAction() : '';
+}, []);
+
+useEffect(() => {
+  document.addEventListener("keydown", undoRedoKey);
+  return () => {
+    document.removeEventListener("keydown", undoRedoKey);
+  };
```
Comment (khuongdn16) [inline app/src/containers/RightContainer.tsx:218]: Please advise why we need to remove the event listener here? It is not neccessary to keep the listener active to listen for the key press?

**Review by khuongdn16 — CHANGES_REQUESTED** [review]
Please see my comments.

**Review by khuongdn16 — APPROVED** [review]
Looks good.

**Review by Anthonytorrero — APPROVED** [review]

**Comment by khuongdn16** [comment]
Approved.

**Pull request merged**